src/lib.rs

```diff
@@ -1535,6 +1535,8 @@ mod impl_internal_constructors;
 mod impl_constructors;
 
 mod impl_methods;
+#[cfg(feature = "std")]
+mod impl_float_maths;
```

Comment:
This line should move down. It should be below `mod numeric`. The order here determines the order of methods in the docs, and my suggestion is down there. We might actually put this inside mod numeric, below impl_numeric?